docs/monoDrive_home/Weather.md

```diff
@@ -21,10 +21,10 @@ conditions and saving them to configuration files.
   </video>
 </div> 
 
-## Adding Weather to a Map
+## Adding Weather to a custom Map
 
 In the monoDrive Scenario Editor, the "Weather" actor can be found in any one of
-the pre-made monoDrive Unreal Engine maps found in the `Content/Maps` directory.
+the pre-made monoDrive Unreal Engine maps.
```

Comment:
```suggestion
the pre-made monoDrive Unreal Engine maps. And stuff.
```

Reply:
@katiemacaulay please see this really cool method that prevents the 3X increase in PR time